waku/waku_sync/common.nim

```diff
@@ -0,0 +1,34 @@
+when (NimMajor, NimMinor) < (1, 4):
+  {.push raises: [Defect].}
+else:
+  {.push raises: [].}
+
+import std/[options], chronos, libp2p/peerId
+import ../waku_core
+
+const
+  DefaultSyncInterval*: Duration = 1.hours
+  WakuSyncCodec* = "/vac/waku/sync/1.0.0"
+  DefaultMaxFrameSize* = 153600 #TODO change to something sensible
```

Comment:
Dumb question: what is this value for?

Reply:
It is used to control the size of the negentropy payload. Lower values leads to more range checks and network round-trips. It's configurable but IDK if we will use anything else than default.